tests/dt.go

```diff
@@ -66,14 +66,14 @@ func (oa *operatorActions) LabelNodes() error {
 		err := wait.PollImmediate(3*time.Second, time.Minute, func() (bool, error) {
 			n, err := oa.kubeCli.CoreV1().Nodes().Get(node.Name, metav1.GetOptions{})
 			if err != nil {
-				klog.Errorf("get node:[%s] failed! error: %v", node.Name, err)
+				k8se2elog.Logf("get node:[%s] failed! error: %v", node.Name, err)
```

Comment:
Why do you use `Logf` to replace `Errorf`? and it seems more usages exist.

Reply:
because there is always "failed" "error" in such logs, but such log is generally not necessary in an e2e test. We should focus on why things goes wrong and fail fast, but not recover from failures in e2e tests.
Also, there's only `Logf` and `Failf` in kubernetes 2e2 framework libarry. I think it follows the same design philosophy.